planning/scenario_planning/common/motion_velocity_optimizer/src/motion_velocity_optimizer.cpp

```diff
@@ -416,18 +451,20 @@ void MotionVelocityOptimizer::calcInitialMotion(
         initialize_type_ = InitializeType::ENGAGING;
         initial_vel = planning_param_.engage_velocity;
         initial_acc = planning_param_.engage_acceleration;
-        ROS_DEBUG(
+        RCLCPP_DEBUG(
+          get_logger(),
           "[calcInitialMotion]: vehicle speed is low (%.3f), and desired speed is high (%.3f). Use "
           "engage speed (%.3f) until vehicle speed reaches engage_vel_thr (%.3f). stop_dist = %.3f",
           vehicle_speed, target_vel, planning_param_.engage_velocity, engage_vel_thr, stop_dist);
         return;
       } else {
-        ROS_WARN_THROTTLE(
-          3.0, "[calcInitialMotion]: stop point is close (%.3f[m]). no engage.", stop_dist);
+        RCLCPP_WARN_THROTTLE(
+          get_logger(), *get_clock(), 3000 /*ms*/,
```

Comment:
correct. But even safer would be
```suggestion
          get_logger(), *get_clock(), (3000ms).count(),
```

Reply:
done. not using literal here, so just use the chrono::milliseconds.